paasta_tools/paasta_metastatus.py

```diff
@@ -66,10 +66,10 @@ def get_extra_mesos_slave_data(mesos_state):
     slaves = dict((slave['id'], {
         'free_resources': slave['resources'],
         'hostname': slave['hostname'],
-    }) for slave in mesos_state['slaves'])
+    }) for slave in mesos_state.get('slaves', []))
```

Comment:
I'm... going to make the call that I think this is wrong.

If we are literally missing the 'slaves' key, that is some sort of other error that we need to raise an exception for.

At this point of the code, it should be considered safe that the mesos_state is correctly formatted. (pretty sure)